charts/nri-kube-events/templates/_helpers.tpl

```diff
@@ -0,0 +1 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
Nit: As the `_helpers.tpl` file is empty (in spite of the vim syntax hint), could it be deleted? (Just asking because sure if not having this file could be a problem)

Reply:
Sure, updated here 417ab3102ba780ebbc61a2747091cf3c379b1d40.